articles/migrations/index.md

```diff
@@ -56,62 +45,68 @@ If you are currently implementing login in your application with Lock v8, v9, or
 
 If you have any questions, create a ticket in our [Support Center](${env.DOMAIN_URL_SUPPORT}).
 
-### Introducing Resource Owner Support for oauth/token Endpoint
+---
+
+## 2018 Upcoming Endpoint Migrations
+
+::: note
+Based on customer feedback, we have adjusted our plans and will continue to maintain and support the below listed proprietary endpoints. We will also publish guidance on the most effective ways to transition your applications to standards-based protocols. In the event that we need to make security enhancements to any of these legacy endpoints, we will promptly announce timeframes and guidelines for any required changes.
```

Comment:
Maybe simplify the language a bit? For example `In the event that we need` can be `If we need`.